Atomic/syscontainers.py

```diff
@@ -1628,7 +1628,7 @@ def _inspect_system_branch(self, repo, imagebranch):
         image_id = commit_rev
         id_ = None
 
-        if len(branch_id) == 64:
+        if SystemContainers.is_hex(branch_id):
```

Comment:
Here we should also check the length of branch id ==64, because there could be a case, that cafe is the image as giuseppe pointed out earlier